hardware_interface/include/joint_limits_interface/joint_limits_interface.hpp

```diff
@@ -45,33 +46,33 @@ class JointLimitHandle
 {
 public:
   JointLimitHandle()
-  : prev_pos_(std::numeric_limits<double>::quiet_NaN()),
-    prev_vel_(0.0),
-    jposh_("position"),
+  : jposh_("position"),
```

Comment:
use standardized names always when possible

```suggestion
  : jposh_(hardware_interface::HW_IF_POSITION),
```